server/models/schema.sql

```diff
@@ -889,6 +889,8 @@ CREATE TABLE `inventory_group` (
   `cogs_account` mediumint(8) unsigned DEFAULT NULL,
   `stock_account` mediumint(8) unsigned DEFAULT NULL,
   `donation_account` mediumint(8) unsigned DEFAULT NULL,
+  `expires` TINYINT(2) DEFAULT 1,
```

Comment:
The `expires` column is supposed to be like a Boolean, it can have only 1 or 0. So, why it has the type `TINYINT(2)` ?

Could you set it to `TINYINT(1) `